src/languages/pf.js

```diff
@@ -0,0 +1,58 @@
+/*
+Language: pf
+Category: config
+Author: Peter Piwowarski <oldlaptop654@aol.com>
+Description: The pf.conf(5) format as of OpenBSD 5.6
+*/
+
+function(hljs) {
+  var MACRO = {
+    className: 'variable',
```

Comment:
Very confused. `var MACRO` contains mode definition with `variable` class, but you specify `macro` in CSS classes reference.